scipy/signal/_waveforms.py

```diff
@@ -261,7 +261,8 @@ def gausspulse(t, fc=1000, bw=0.5, bwr=-6, tpr=-60, retquad=False,
         return yI, yQ, yenv
 
 
-def chirp(t, f0, t1, f1, method='linear', phi=0, vertex_zero=True):
+def chirp(t, f0, t1, f1, method='linear', phi=0, vertex_zero=True,
+          complex=False):
```

Comment:
New params should be keyword only.
```suggestion
def chirp(t, f0, t1, f1, method='linear', phi=0, vertex_zero=True, *,
          complex=False):
```